sling/nlp/document/lexical-encoder.cc

```diff
@@ -313,6 +313,34 @@ int LexicalFeatures::InitWordEmbeddings(const string &filename) {
   return found;
 };
 
+namespace {
+
+// Adds to 'trace', 'num' feature values located at 'feature' in 'data'.
+// Ignores -1, i.e. invalid lexical feature values.
+void AddTrace(int token, const string &name, int num,
+  myelin::Instance &data, myelin::Tensor *feature, Trace *trace) {
+  if (feature == nullptr) return;
+  const auto *location = data.Get<int>(feature);
```

Comment:
What about const int * instead of auto? The name 'location' is also a bit confusing to me.